src/lib/connection.js

```diff
@@ -270,6 +271,13 @@ export default class Connection extends EventEmitter {
     this.removeAllListeners()
   }
 
+  sendPing() {
+    if (this.connected === true) {
+      this.socket.ping("");
```

Comment:
The client uses `isomorphic-ws` that uses `ws` in Node and browser's native WebSocket. However there is no `ping` in browsers: https://developer.mozilla.org/en-US/docs/Web/API/WebSocket

At the minimum this should be pointed out in README.